<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_svg_datawindow</class>
 <widget class="QWidget" name="dlg_svg_datawindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据窗口选择</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ssp_gui.qrc">
    <normaloff>:/ssp_gui/Resources/NOTE06.ICO</normaloff>:/ssp_gui/Resources/NOTE06.ICO</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,3,0">
       <item>
        <widget class="QComboBox" name="cmbClsName">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtFilter">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbName">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnCfg">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="ssp_gui.qrc">
           <normaloff>:/ssp_gui/Resources/cfg.gif</normaloff>:/ssp_gui/Resources/cfg.gif</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="twDataset">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ssp_gui.qrc"/>
 </resources>
 <connections/>
</ui>
